navigation_task_node:
  ros__parameters:
    map_origin_lat: -33.72242824301795
    map_origin_lon: 150.6740063854522
    map_origin_set: true
    gps_start_lat: -33.722553423946906
    gps_start_lon: 150.67394210456843
    gps_end_lat: -33.72253671663162
    gps_end_lon: 150.67413245758723
    gps_start_x: 0.0
    gps_start_y: 0.0
    gps_end_x: 1.1791211357704299
    gps_end_y: 17.485397720327114
    gps_frame_coords_set: true
    map_origin_topic: "/map/origin"
    odom_topic: "/seapath/odom/ned"
    landmark_topic: "landmarks_out"
    assignment_confidence: 10  # Number of consequtive identical assignments from auction algorithm before we consider the assignment as correct
    
    # Task specific parameters
    distance_to_first_buoy_pair: 2.0 # Distance in x-direction to first buoy pair in meters from current postition (not gps) position in base_link frame
    # If the distance is greater than 6.0 we drive distance-4m towards the first buoy pair before trying to find it